WORKSPACE

```diff
@@ -597,3 +597,32 @@ new_local_repository(
         visibility = ["//visibility:public"],)""",
     path = "empty",
 )
+
+# Dev tools
+http_file(
```

Comment:
This will not be fun to maintain manually (if you look at `cargo_bazel_linux_x64` and other variants I've been lazy with upgrading ones other than my platform). I think this can go ahead as-is though, #2775 will eventually make this easier.